### ISSUE TITLE
[Elastic Agent] Metricbeat logging is duplicating every event

### ISSUE THREAD

**Issue by mostlyjason** [elastic/beats#23310]
Elastic Agent's metricbeat logs is duplicating the entire body of each metricbeat message in plaintext format. This eats up a lot of storage space and is not necessary since the metrics are already stored using the metrics data type. Its so inefficient its using several times more space than my overall system metrics. Is it possible to turn these logs off by default, perhaps by setting the log level to debug?

Screenshot showing 5.9GB of space usage after 2-3 days:
![image](https://user-images.githubusercontent.com/6370704/103244285-60aaa580-4911-11eb-8f9e-fd690f564a38.png)

Example log event to logs-elastic_agent.metricbeat-default:
```
Publish event: { "@timestamp": "2020-12-28T21:34:46.132Z", "@metadata": { "beat": "metricbeat", "type": "_doc", "version": "7.10.1", "raw_index": "metrics-system.process-default" }, "service": { "type": "system" }, "system": { "process": { "cmdline": "/opt/Elastic/Agent/data/elastic-agent-1da173/install/filebeat-7.10.1-linux-x86_64/filebeat -E setup.ilm.enabled=false -E setup.template.enabled=false -E management.mode=x-pack-fleet -E management.enabled=true -E logging.level=debug -E logging.level=debug -E http.enabled=true -E http.host=unix:///tmp/elastic-agent/default/filebeat/filebeat.sock -E logging.json=true -E logging.ecs=true -E logging.files.path=/opt/Elastic/Agent/data/elastic-agent-1da173/logs/default -E logging.files.name=filebeat-json.log -E logging.files.keepfiles=7 -E logging.files.permission=0640 -E logging.files.interval=1h -E path.data=/opt/Elastic/Agent/data/elastic-agent-1da173/run/default/filebeat--7.10.1", "memory": { "size": 1497452544, "rss": { "bytes": 68112384, "pct": 0.0109 }, "share": 19234816 }, "cpu": { "start_time": "2020-12-25T12:09:40.000Z", "total": { "pct": 0.0039, "norm": { "pct": 0.002 }, "value": 1.79456e+06 } }, "state": "sleeping", "fd": { "open": 20, "limit": { "soft": 1024, "hard": 524288 } } } }, "user": { "name": "root" }, "elastic_agent": { "id": "e961ad1e-ccfe-4702-b9cc-69fe6a2c5ea1", "snapshot": false, "version": "7.10.1" }, "ecs": { "version": "1.6.0" }, "process": { "pgid": 14012, "working_directory": "/opt/Elastic/Agent/data/elastic-agent-1da173/install/filebeat-7.10.1-linux-x86_64", "executable": "/opt/Elastic/Agent/data/elastic-agent-1da173/install/filebeat-7.10.1-linux-x86_64/filebeat", "args": [ "/opt/Elastic/Agent/data/elastic-agent-1da173/install/filebeat-7.10.1-linux-x86_64/filebeat", "-E", "setup.ilm.enabled=false", "-E", "setup.template.enabled=false", "-E", "management.mode=x-pack-fleet", "-E", "management.enabled=true", "-E", "logging.level=debug", "-E", "logging.level=debug", "-E", "http.enabled=true", "-E", "http.host=unix:///tmp/elastic-agent/default/filebeat/filebeat.sock", "-E", "logging.json=true", "-E", "logging.ecs=true", "-E", "logging.files.path=/opt/Elastic/Agent/data/elastic-agent-1da173/logs/default", "-E", "logging.files.name=filebeat-json.log", "-E", "logging.files.keepfiles=7", "-E", "logging.files.permission=0640", "-E", "logging.files.interval=1h", "-E", "path.data=/opt/Elastic/Agent/data/elastic-agent-1da173/run/default/filebeat--7.10.1" ], "name": "filebeat", "pid": 14033, "ppid": 14012 }, "event": { "dataset": "system.process", "module": "system", "duration": 386856091 }, "metricset": { "name": "process", "period": 10000 }, "data_stream": { "type": "metrics", "dataset": "system.process", "namespace": "default" }, "host": { "os": { "family": "debian", "name": "Debian GNU/Linux", "kernel": "4.19.0-13-amd64", "codename": "buster", "platform": "debian", "version": "10 (buster)" }, "id": "993c43f9133a4d559e52875176086500", "containerized": false, "ip": [ "192.168.2.39", "10.147.17.250", "172.17.0.1", "172.19.0.1", "172.20.0.1" ], "hostname": "monitor", "architecture": "x86_64", "name": "monitor" }, "agent": { "name": "monitor", "type": "metricbeat", "version": "7.10.1", "hostname": "monitor", "ephemeral_id": "1e17b5ef-c84f-4f38-bce2-21419c679052", "id": "53def08a-b33c-49e8-ba23-abdf4e0029c9" } }
```

**Comment by ph** [comment]
That seems a weird but to me, @mostlyjason did you test with 7.10 or 7.11? 

Because I think the log statement that you are seeing is from 7.10 and previous, we have made a change to log level and allow the log level to be modified remotely.

@michalpristas going to assign it to you so you can confirm.

**Comment by mostlyjason** [comment]
Thanks I'm using version 7.10.1

**Comment by michalpristas** [comment]
yes these should be debug logs, we increased default level to INFO in 7.11

**Comment by ph** [comment]
Fixed in 7.11
